src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/TestGeneralOperators.java

```diff
@@ -171,4 +172,105 @@ public void testInvalidInputSizeIsNotNullSmall() throws EvaluationException {
         Operators.NULLABLE_ISNOTNULL.evaluate(new IVariable[]{});
     }
 
+    /**
+     * ----------------------------------- ABS -----------------------------------
```

Comment:
Can you also add `testInvalidInputSizeIAbsLarge` and `testInvalidInputSizeAbsSmall` tests?
(For consistency with the unit tests for the other operators)